pages/trust-admin.js

```diff
@@ -182,11 +187,18 @@ export const getServerSideProps = propsWithContainer(
     const retrieveWardVisitTotalsStartDateResponse = await container.getRetrieveWardVisitTotalsStartDateByTrustId()(
       authenticationToken.trustId
     );
+    const retrieveReportingStartDateResponse = await container.getRetrieveReportingStartDateByTrustId()(
+      authenticationToken.trustId
+    );
 
     const wardVisitTotalsStartDate = retrieveWardVisitTotalsStartDateResponse.startDate
       ? formatDate(retrieveWardVisitTotalsStartDateResponse.startDate)
       : null;
 
+    const reportingStartDate = retrieveReportingStartDateResponse.startDate
+      ? formatDate(retrieveReportingStartDateResponse.startDate)
```

Comment:
Would it make sense to format the date inside the usecase instead of the page? Or do we want to keep that logic separate?

Reply:
When doing the original booking visits start date, my thoughts were formatting the date was a presentational thing so kept it on the page. But looking now and after we moved the moment duration code to the average visit time usecase, it just feels like noise on this page that could be moved to the usecase...